config/application.rb

```diff
@@ -56,6 +56,9 @@ class Application < Rails::Application
     config.proposal_mode = false
     config.proposal_mode = true if ENV['proposal_mode'] == 'true'
 
+    #Feature toggle for using legacy order service
+    config.use_legacy_order_service = ENV['use_legacy_order_service'] == 'true'
+
```

Comment:
Note, this will require us to set SIT by hand to false (which should be easy) but will need to be a step taken when deployed.